v1.x/batch/index.md

```diff
@@ -26,7 +26,58 @@ Name | Type | Description
 `success` | `boolean` | Whether the request succeeded or failed. <br> **Example**: `true`
 `message` | `string` | If the request was not successful, this contains the error message. <br> **Example**: `Unknown fatal error occurred.`
 
-### Main URL Construction
+### Basic Construction
+
+The steps to creating a batch request are:
+
+1. Encode the sub URLs for each request in the batch. Then, do one of the following:
+2. For a GET, append the requests to the main URL and md5() the whole thing.
+3. For a POST, put the requests in the POST data and construct the main URL like normal.
+
+### Sub URL Construction
+
+The sub URL contains multiple single-call URLs.
+
+One sub URL would look similar to this:
+
+```
+requests[]=/data-store/?game_id=32&key=test&signature=912ec803b2ce49e4a541068d495ab570
+```
+
+It is constructed the same way as a single-call URL, but with the URL of the Game API and the API version removed:
+
+```
+"requests[]=" + urlencode( "/data-store/get/?username=test&user_token=test-token&key=test&game_id=456" + "&signature=" + md5( "/data-store/get/?username=test&user_token=test-token&key=test&game_id=456" + privateKey ) )
+```
+
+The first sub URL starts with `requests[]=` and all others start with `&requests[]=`.
```

Comment:
We should start this with something like, "We add the sub URL by attaching it to the requests[] query field name." I don't think we should mention that the first doesn't have an &. I think instead we should mention how to construct, and then afterwards we say something like, "After you have all your sub URLs, you join them together with an `&` character.